add-collateral.json

```diff
@@ -1 +1 @@
-{"50":{"tokenAddress":"0x8f9920283470f52128bf11b0c14e798be704fd15","testOracle":"0x0000000000000000000000000000000000000000","fathomProxyFactory":"0x0acf88Ba60561372Ef93a67E04e1A55d9fC1D910","fathomProxyAdmin":"0xb82AD475fc113671840D510B60Cbae5630a07f3B","fixedSpreadLiquidationStrategy":"0xfe5037504E0EF5eC2DfBEEA03f9d9cB43580EF23"},"51":{"tokenAddress":"0x97EC6730Fd5F138fCB167cb62A9a4c1A8Be2eD7d","testOracle":"0x0000000000000000000000000000000000000000","fathomProxyFactory":"0xf62f41d00EAaf02fD31FA24C5630C9AcfEf8D69F","fathomProxyAdmin":"0x6B33e88ec42291Bf82C9E7CE7f45DdC3464897bf","fixedSpreadLiquidationStrategy":"0x4bEb8638AFe2892Fa7D46303F60f22e106C9AB7B"},"31337":{"tokenAddress":"0x86504933bb9d24Ea2fECA6679836BBC07570b08D","testOracle":"0x335bEBc68b1d249E0Db8E20990255b2Ff985CdDf","fixedSpreadLiquidationStrategy":"0x0000000000000000000000000000000000000000","fathomProxyFactory":"0x31A3c2fC47dD591172D3067243AE4209BE495346","fathomProxyAdmin":"0xf5CDb34e1284fA7E927CeCDb62C602ea2dBD4E15"},"token":"GLD"}
\ No newline at end of file
+{"50":{"tokenAddress":"0x8f9920283470f52128bf11b0c14e798be704fd15","testOracle":"0x0000000000000000000000000000000000000000","fathomProxyFactory":"0x0acf88Ba60561372Ef93a67E04e1A55d9fC1D910","fathomProxyAdmin":"0xb82AD475fc113671840D510B60Cbae5630a07f3B","fixedSpreadLiquidationStrategy":"0xfe5037504E0EF5eC2DfBEEA03f9d9cB43580EF23"},"51":{"tokenAddress":"0x97EC6730Fd5F138fCB167cb62A9a4c1A8Be2eD7d","testOracle":"0x0000000000000000000000000000000000000000","fathomProxyFactory":"0xf62f41d00EAaf02fD31FA24C5630C9AcfEf8D69F","fathomProxyAdmin":"0x6B33e88ec42291Bf82C9E7CE7f45DdC3464897bf","fixedSpreadLiquidationStrategy":"0x4bEb8638AFe2892Fa7D46303F60f22e106C9AB7B"},"31337":{"tokenAddress":"0x5C9c4C2d825c4Af9967F240Aa40d12E51E60CD2A","testOracle":"0x1386243695a818845809AB9AaC2396123608Ff37","fixedSpreadLiquidationStrategy":"0x0000000000000000000000000000000000000000","fathomProxyFactory":"0x4d09164d6016344B0bEAE55CE44FeE331ce7297F","fathomProxyAdmin":"0xFB8f4a197e59a5c47B3E5c2834c864a9db802F54"},"token":"GLD"}
```

Comment:
one more thing since we are so close to make it 100% work for everyone. maybe include this file with comments into setup readme? and exclude from repo and add to gitignore?